recipes-init/openrc/openrc_0.54.2.bb

```diff
@@ -98,6 +99,7 @@ RDEPENDS:${PN} = " \
     util-linux-mount \
     util-linux-umount \
 "
+RDEPENDS[capabilities] += "libcap"
```

Comment:
No need for this.  I assume that openrc will link against libcap, which means that when you put libcap into DEPENDS in `PACKAGECONFIG[capabilities]` above, by listing it third, bitbake will be smart enough to also pick up that it needs to be in RDEPENDS.

If openrc doesn't link against libcap, then you need to expand PACKAGECONFIG: `PACKAGECONFIG[capabilities] = "-Dcapabilities=enabled,-Dcapabilities=disabled,libcap,libcap"`.  The fourth item here is adding it to RDEPENDS.  https://docs.yoctoproject.org/ref-manual/features.html#ref-features-distro

Reply:
nice! done.